code/index.html

```diff
@@ -4,27 +4,127 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Imbue:wght@500&family=PT+Sans+Narrow:wght@700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+<!---------1. HEADER, LOGO, NAVIGATION---------->
+    <header class="header">
+      <img class="logo" src="./img/logo.png" alt="logotype">
+      <nav class="navigation">
+        <a href="#">
+        <div class="navigation-text">About us</div>
+      </a>
+      <a href="#"> 
+      <div class="navigation-text">Join us</div>
+      </a> 
+      <a href="#">    
+      <div class="navigation-text">Schedule</div>
+    </a>
+      </nav>
+    </header>
```

Comment:
Remember to indent correctly, and possible omit the `div`s here for a cleaner structure. Try styling the links directly :)